src/middlewared/middlewared/plugins/pool.py

```diff
@@ -55,6 +55,7 @@
 ]
 ZPOOL_CACHE_FILE = '/data/zfs/zpool.cache'
 ZPOOL_KILLCACHE = '/data/zfs/killcache'
+TRUENAS_DB = '/data/freenas-v1.db'
```

Comment:
`from middlewared.utils.db import FREENAS_DATABASE`

Reply:
That doesn't exist on 13. Furthermore, the product isn't called freenas anymore